pom.xml

```diff
@@ -951,8 +971,41 @@
                 <jdk>9</jdk>
             </activation>
             <properties>
-                <surefire.argLine.extension>--add-modules ${jigsaw.modules}</surefire.argLine.extension>
+                <maven-surefire-plugin.argLine.add-modules>--add-modules=${maven-surefire-plugin.jigsaw.modules}</maven-surefire-plugin.argLine.add-modules>
+                <arquillian.javaVmArguments.add-opens>
```

Comment:
I took them from the WF patch referenced by Tomaz. I think it's better to be in line with their expectations. I didn't play hide and seek for this one.